src/common/reducers/repositories.js

```diff
@@ -20,14 +20,14 @@ export function repositories(state = {
         ...state,
         isFetching: false,
         error: null,
-        ids: union(state.ids, action.payload.result),
-        pageLinks: action.payload.pageLinks
+        ids: union(state.ids, action.payload.response.result),
+        pageLinks: action.payload.response.pageLinks
       };
     case ActionTypes.REPOSITORIES_FAILURE:
       return {
         ...state,
         isFetching: false,
-        error: action.payload,
+        error: action.payload.error,
```

Comment:
With `callApi` middleware we need to nest error in payload instead of error just being the payload?

I guess it's needed so we can make sure we also pass any other payload items if needed, right?

It breaks a little bit Flux Standard Actions pattern, but I guess it's a compromise we need to make. I think we were already breaking FSA in this regard to send both `id` and `error` in payload. So I guess it needs to stay like that.

So I guess in our FSA 'variant' we should assume that if `action.error` is true then `action.payload.error` should be an error object.